### DISCUSSION TITLE
airdcpp-share-monotor on mapped networkdrives

### DISCUSSION THREAD

**Discussion by vildsvin666** [airdcpp/airdcpp-windows#168] [General]
When mapping a networkdrive from linux to my windows server 2016 share-monotor don't find changes in share. Same for networkdrive connected with sshfs-win share. Is this a bug or is it the same on mapped drives from windows? Dont got any more computer to test this.

The scenario:
Win server with Airdc 4.21. I share folder TEMP from my ubuntu computer wicth the windows server has as a network drive "Z".
When adding or remove folders in TEMP the share doesent update automatic, tried all settings in share-monitor "count the last modification time". When adding / remove folders on the local drive it works as expected.

Monitoring the networkdrive I now use  refreshing options "Incoming refresh time 2mins" witch works.

**Comment by maksis** [comment]
This should be posted to https://github.com/airdcpp-web/airdcpp-share-monitor/issues

**Reply by vildsvin666** [reply]
Yes I suspected that but want to know how it works with windos shares or not. Or if someone else got the same issue.

**Reply by vildsvin666** [reply]
Seems like I got it to work, permissions issues... So share monitor works fine :)
